packages/gatsby/src/bootstrap/index.js

```diff
@@ -321,7 +320,7 @@ module.exports = async (args: BootstrapArgs) => {
   activity.end()
 
   // Collect resolvable extensions and attach to program.
-  const extensions = [`.js`, `.jsx`]
+  const extensions = [`.js`, `.jsx`, `.mjs`, `.wasm`, `.json`]
```

Comment:
I'm a little surprised we didn't already support JSON 😅 

Also do we need to add loaders to actually do something useful with these, or are we presuming that some plugin would need to be installed to handle them?

Reply:
As per this comment https://github.com/graphql/graphql-js/issues/1272#issuecomment-377384574, shouldn't `.mjs` appear before `.js`?

Reply:
Yeah @kkemple wised me up to this and I fixed it in https://github.com/gatsbyjs/gatsby/pull/8339

Reply:
Thanks!